build.sbt

```diff
@@ -28,7 +28,7 @@ libraryDependencies ++= Seq(
   "com.typesafe.play" %% "play-json" % "2.7.3",
   "info.debatty" % "java-string-similarity" % "1.2.1",
   "org.slf4j" % "slf4j-api" % "1.7.26",
-  "org.slf4j" % "slf4j-nop" % "1.7.26" % Test
+  "org.slf4j" % "slf4j-nop" % "1.7.26" % Test,
```

Comment:
Is this change needed?

Reply:
I tested some dependency's and forget to remove it. If it's problem, I will remove it